### DISCUSSION TITLE
How to set the patch_size and the final_patch_size?

### DISCUSSION THREAD

**Discussion by afeizqf** [MIC-DKFZ/nnUNet#694] [Q&A] [answered]
Hi There,

Thank you for sharing "nnUNet" on GitHub.

I want to know how to set the patch_size and the final_patch_size by myself.

Thanks.

**Comment by FabianIsensee** [comment, marked as answer]
patch_size you dont need to worry about since that is determined based on final_patch_size

final_patch_size is in the plans files after plan_and_preprocess. Have a look at the tutorials: https://github.com/MIC-DKFZ/nnUNet/tree/master/documentation/tutorials

**Reply by afeizqf** [reply]
Thanks!
